x-pack/plugins/metrics_data_access/server/lib/metrics/index.ts

```diff
@@ -42,7 +42,7 @@ export const query = async (
   const filter: Array<Record<string, any>> = [
     {
       range: {
-        [options.timerange.timeFieldName || TIMESTAMP_FIELD]: {
+        [TIMESTAMP_FIELD]: {
```

Comment:
I saw in the metric threshold rule document, that there is this option to [specify timestamp](https://www.elastic.co/guide/en/observability/current/metrics-threshold-alert.html#metrics-alert-settings) which I didn't see it in the setting, did we have this setting at some point?

![image](https://github.com/elastic/kibana/assets/12370520/b9951b8a-3bbe-4bab-b1fb-b6bab0fa2648)

Reply:
@neptunian do you have any info on this ?